selenium_tests/course_info/page_objects/course_people_page_object.py

```diff
@@ -67,11 +77,38 @@ def select_role_type(self, canvas_role):
         self.find_element(*Locators.ROLES_DROPDOWN_LIST).click()
         self.find_element(By.LINK_TEXT, canvas_role).click()
 
-    def add_was_successful(self):
+    def single_add_was_successful(self):
         # Verify success text
-        # todo: this does not check _which_ add was successful...
+        # todo: we can refactor this; see "add_is_successful_by_id" method
+        try:
+            self.find_element(*Locators.SINGLE_ADD_ALERT_TEXT)
+        except NoSuchElementException:
+            return False
+        return True
+
+
+    def add_is_successful_by_id(self, user_id):
+        # Verify user ID is displayed
+        try:
+            self.find_element(*CourseInfoBasePageObjectLocators.TD_TEXT_XPATH(
```

Comment:
Indeed, this is checking against the course members list that appears on the page (not the results table).  It's a little bit of a more precise check than just the alert text, but you're right regarding the pagination.   There were a bunch of TODOs carryover in the code for the single create that "the test methods weren't actually checking on actual ID", so I thought I get my practice in on having a method that checks for ID (as a reference for future if needed).  

We could go either way really.  Considering most of tests does the cleanup, it may not be an issue?  I had already added a note in the multi_user_add_successful test:   re: "''' Limitation:  This may no longer be a valid test if there are a lot of ID in the test course and the user being added appears on second page due to pagination.  If that is the case, we could go back to checking on alert text'''" 

Let me know if you have particularly strong feelings about using alert text instead (due to potential flakiness factor, if that's the biggest concern) and I could modify it.  Thanks!